Makefile

```diff
@@ -104,17 +100,17 @@ healpix_bare/healpix_bare.o : healpix_bare/healpix_bare.c
 pg_version := $(word 2,$(shell $(PG_CONFIG) --version))
 has_support_functions = $(if $(filter-out 9.% 10.% 11.%,$(pg_version)),y,n)
 
-crushtest: REGRESS += $(CRUSH_TESTS)
+crushtest: TESTS += $(CRUSH_TESTS)
```

Comment:
This change leaves out `init` from `crushtest`, right? Why would we want to do that?

Reply:
crushtest is unaffected by this change, it runs the normal "installcheck" target, just with more tests. The list of tests is in $(TESTS) now, $(REGRESS) is only used internally by the pgxs rules that invoke pg_regress.

Adding to REGRESS would also work for this use case, but all the other rules in the Makefile need to add to TESTS so "make test" is also supported, so I think it's cleaner to have crushtest append to TESTS as well.

Reply:
Sorry, where is the `installcheck` target defined? I can't find it.

Reply:
In PostgreSQL's pgxs.mk. The tests are run on `$(REGRESS)`, our Makefile defines `REGRESS     = init $(TESTS)`, crushtest appends to `TESTS`, so when `crushtest` is run, it includes the crush files.

Reply:
Well, OK then!